src/components/SearchFilterList.jsx

```diff
@@ -1,8 +1,15 @@
-import { useState } from "react"
+import { useState,useContext } from "react"
+import RecipeContext from "../context/RecipeContext"
 
-function SearchFilterList({title,filter,setType}) {
+function SearchFilterList({title,filter,type}) {
 
     const [active,setActive] = useState(false)
+    const {urlEndpoints,setUrlEndpoints} = useContext(RecipeContext)
+    
+    const handleSetUrlEndpoints = (arr)=>{
+        //@QUESTION how do i pass type dynamically to set properties of urlEndpoints based on type passed to component?
```

Comment:
At this point, is there a way to use the type prop passed to SearchFilterList component from SearchFilter to set the urlEndpoints state.